scripts/surface_charge/surface_charge.py

```diff
@@ -0,0 +1,270 @@
+#
+# This script can be used for any purpose without limitation subject to the
+# conditions at http://www.ccdc.cam.ac.uk/Community/Pages/Licences/v2.aspx
+#
+# This permission notice and the following statement of attribution must be
+# included in all copies or substantial portions of this script.
+#
+# The following line states a licence feature that is required to show this script in Mercury and Hermes script menus.
+# Created 18/08/2024 by Alex Moldovan (https://orcid.org/0000-0003-2776-3879)
```

Comment:
Is `# ccdc-licence-features not-applicable` missing here? I actually don't know when it is needed...